wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/MotorControllerGroup.java

```diff
@@ -71,6 +71,14 @@ public double get() {
     return 0.0;
   }
 
+  @Override
+  public double getVoltage() {
```

Comment:
We don't need this; it can just use the inherited impl from MotorController which delegates to the already-existing `get()`